codelabs/custom_generator/custom_generator.md

```diff
@@ -153,43 +153,28 @@ A language generator has a single entry point: `workspaceToCode`. This function
 
 The first step is to define and call your language generator.
 
-A custom language generator is simply an instance of `Blockly.Generator`. Call the constructor, passing in your generator's name, and store the result.
+A custom language generator is simply an instance of `Blockly.Generator`. In `custom_generator.js` call the constructor, passing in your generator's name, and store the result.
 
 ```js
 const codelabGenerator = new Blockly.Generator('JSON');
 ```
 
 ### Generate code
 
-Add a button to the playground to generate JSON:
-
-```html
-<input type="button" value="To JSON" onclick="toJSON()">
-```
-
-Implement `toJSON` so that it generates code and outputs it to two places: the console, and the playground's text area.
+Next, register your new generator with the playground. Find the `configurePlayground` function in `advanced_playground.html` and add this line:
```

Comment:
Not exactly a recommendation, but do you think some people might get lost at where to put the code above (initialization) if the code below is going in configurePlayground.

If I'm following right, both of them should go in configurePlayground.

Reply:
Yeah I had that code going in `custom_generator.js`, so obviously we have some confusion here.

I prefer having it in custom_generator.js so that it hangs out with the rest of the generator definition.